sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -135,9 +135,8 @@ export type EntityStatus = "Active" | "Creating" | "Deleting" | "ReceiveDisabled
 export interface GetMessageIteratorOptions extends OperationOptionsBase {
 }
 
-export { isMessagingError }
-
-export { MessageErrorCodes }
+// @public
+export function isServiceBusError(err: Error | AmqpError | ServiceBusError): err is ServiceBusError;
```

Comment:
When would a user ever encounter an `AmqpError`?

Reply:
I'll remove it and see - I think there was an internal case where AmqpError was passed in and it's not derived from Error.

Reply:
Ah, it's for when we call translate on the error that comes back in the EventContext (from rhea itself). If AmqpError was only derived from Error this wouldn't be an issue.

Thoughts?

Reply:
If it is only for internal case, then we will need to figure out a way for the external facing method to not include it. Because from a user's perspective, this can be confusing.

Reply:
Can we have two separate methods? One would be public and be used by customers and the other for us?

Reply:
Logged https://github.com/Azure/azure-sdk-for-js/issues/12264 to follow up